README.md

```diff
@@ -1 +1,61 @@
-# Microverse_Week_rpt
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio: mobile version
+
+> This is the Third Project and i have added a portfolio section showing the projects worked on. 
+
+
+This is a Portfolio that works as a presentation card for a Software developer, showcasing some important projects, personal and contact info, while keeping a fresh and elegant web design. 
+
+## Built With
+
```

Comment:
## [OPTIONAL]
- [ ] I think your project would look more professional if you include a screenshot of the home first section. This will help visitors have an idea of what is the content of the project(an image).